renovate.json

```diff
@@ -7,5 +7,26 @@
     ],
     "ignorePaths": [
       ".pre-commit-config.yaml"
-    ]
+    ],
+    "docker": {
+        "enabled": true,
+        "postUpdateOptions": [
+            "echo {{{newValue}}} > .baseimagedigest",
+            "docker run --rm --entrypoint sh -u 0 quay.io/redhat-services-prod/insights-management-tenant/insights-floorist/floorist-operator:latest -c 'dnf -y upgrade > /dev/null; rpm -qa | sort | sha256sum' >> .baseimagedigest"
```

Comment:
Are the images `quay.io/redhat-services-prod/insights-management-tenant/insights-floorist/floorist-operator` tagged as `latest`? Can you check when those are tagged as `latest`?

Is there a podman version?

Reply:
Yes they are tagged as `latest`.  They're tagged if the `on-push` pipeline succeeds in building an image with no erroneous steps. (if all tasks of the `.tekton/floorist-operator-push.yaml` pass)

This is only done after merging a commit to the `master` branch of this repo. Yesterday at 12:04 I've merged a Konflux update and the image in the repo appeared 12:14.

The manager is called docker, so no, there's no `podman` version. It's just the piece of automation that takes care of container images.

Reply:
I will try to look for a way to test the added manager and it's `after` steps. Not sure if this is possible at the moment.

Reply:
I haven't found any way to run the updated config. There's only a [validator for the new config](https://docs.renovatebot.com/getting-started/installing-onboarding/#reconfigure-via-pr).

The validator has passed so I will merge the config change and hope it will work as I expect it to...